### PR TITLE
Added some guidance on translating media labels, closes #153

### LINKED ISSUE #153: Add information on how to handle the media files for multiple language forms
Under the **Media** heading, Add that in multi-language forms, you have to specify media files for each included language. 

It is hinted to under **Multiple language support,** but could save users a lot of frustrations if it was also directly written under **Media**.

### PR DESCRIPTION
**Pull request by MartijnR**

### CODE REVIEW

**Comment by jnm** [comment]
Hi @MartijnR, I wouldn't mind having a look at them, but I don't have access to do anything beyond commenting. @tinok might have the appropriate privileges but is probably short on time.

**Comment by MartijnR** [comment]
Thanks @jnm! I sent an invite, and I think I can request reviews after you've accepted.

**Comment by MartijnR** [comment]
@jnm, just pinging as a reminder, in case you can review (and the other 3 PRs perhaps too...) :)

File: _sections/home-english.md [hunk 1]
```diff
@@ -534,20 +534,23 @@ Some platforms also support dynamic repeat counts.  In the example below, the nu
...
 | integer           | age         | How old are you?    | ¿Cuántos años tienes? | . <= 150   |
 | ================= | =========== | ============        | ============          | ========== |
 | survey            |             |                     |                       |            |
 
-Form language and user interface language may be the determined separately by the application and may not match. To facilitate matching both (in the future), it is recommended, though optional, to add a 2-character language code after the language name. The official 2-character language codes, called _subtags_ are published [here](http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) (search the page with Ctrl-F or Cmd-F).
-
+You can also add a different language column for hints and media files by using the same ``::language (code)`` construct. See the example below and also see the [XLSForm standard document](https://docs.google.com/spreadsheet/ccc?key=0AjZ4hMHTat-YdFZSY3BOZWtGeTdxWGQ1clZoVEZkamc&usp=sharing) to see exactly what kinds of column headers can accept a language modification.
```
[the remark this answers is not in the document]
Comment (lognaturel) [reply _sections/home-english.md]: Agree with these suggestions. @MartijnR do you want to update so this can be merged?

**Comment by MartijnR** [comment]
Ah thanks. I didn't see this review!

**Comment by lognaturel** [comment]
Trying to do some spring cleaning today! Thanks, @MartijnR!

**Pull request merged**